src/app/content/components/Page.tsx

```diff
@@ -92,17 +94,33 @@ export class PageComponent extends Component<PropTypes> {
   }
 
   public render() {
-    return <BookStyles>
-      {(className: string) => <MainContent className={[this.props.className, className].join(' ')}>
-        <div data-type='chapter'>
-          <div
-            data-type='page'
-            ref={(ref: any) => this.container = ref}
-            dangerouslySetInnerHTML={{ __html: this.getCleanContent()}}
-          />
-        </div>
-      </MainContent>}
-    </BookStyles>;
+    const book = this.props.book && BOOKS[this.props.book.id] || null;
+    if (book && book.bookStyleName) {
+      const bookStyles = assertString(getBookStyles().get(book.bookStyleName),
+        `missing book style: "${book.bookStyleName}"`);
+      // tslint:disable-next-line:variable-name
+      const BookStyles = styled.div`
```

Comment:
is this whole top block only here to load the intro-business styles that we've been historically using? is this a real life use-case? defining a styled component like this is a performance no-no since it re-processes it on every render. also the wrapping divs for chapter, page are not required for any particularly good reason, just that they were already in the css.

if we're gonna need separate stylesheets for different books, i'd rather build that on the new system you put in place, and not have all this junk.

Reply:
Hmm, ok. I can try to switch cnx-recipes so that it always provides a style (either generic or book-specific). It seems like this code will still need to be somewhat dynamic since it would have to ask `cnx-recipes` for the book style (by providing the book slug name)

Reply:
i thought the point of these styles was that they wouldn't involve cnx-recipes? can we just put book specific css files in rex? can we rip out all the cnx-recipes integration until we do the baked styles down the road?